src/syft/core/nodes/device/network.json

```diff
@@ -0,0 +1,25 @@
+{
```

Comment:
Syft shouldn't have user-defined json files. This should be a class with default parameter values and the ability to (optionally) be created from a static file. This will allow PyGrid to create config objects from files, but PySyft will simply run with good defaults.

Reply:
the good default for surrounding workers and specs will probably just be: I don't know anyone else (I'm an isolated worker) and I have no clue what my specs are, but I'll run anyway.

do you see it differently?

Reply:
I see it the same way. Functionally we're on the same page... literally just the difference between setting defaults in a code object instead of a .json file.

PyGrid can have its own default .json if it wants. I'm just against it in PySyft.

Reply:
makes sense